### DISCUSSION TITLE
Custom system clock

### DISCUSSION THREAD

**Discussion by yellow-flash02** [rafaelvcaetano/melonDS-android#1242] [Ideas]
currently it uses the default system clock it would be nice if we had option to change date and time to our liking in emulator itself for time based events and many other things

**Comment by rafaelvcaetano** [comment]
This has not yet been implemented because the PC version also does not have it. But they have already implemented it for the next release ([see here](https://melonds.kuribo64.net/comments.php?id=192)), and once it's available, I will port the feature to the Android version.

**Reply by yellow-flash02** [reply]
There's option in drastic too but ig they aren't open source so u cant really use that either ways Thanks alot man u are gamechanger for years people avoided the lack of  multiplayer in DS emulators u did it much appreciated.
